integration/pull_test.go

```diff
@@ -872,3 +872,134 @@ func TestFullLayerRead(t *testing.T) {
 		t.Fatalf("Expected 100%% fetchedPercent, found %v", layers.FetchedPercent)
 	}
 }
+
+func TestPullWithParallelism(t *testing.T) {
+	img := rabbitmqImage
+
+	fetchCases := []struct {
+		name string
+		opts []snapshotterConfigOpt
+	}{
+		{
+			name: "no parallel fetching",
+		},
+		{
+			name: "set chunk size",
+			opts: []snapshotterConfigOpt{
+				func(cfg *config.Config) {
+					cfg.PullModes.Parallel.ConcurrentDownloadChunkSize = 1_000_000
```

Comment:
need to explicitly specify "containerd" content store for these tests.

Reply:
Done